### DISCUSSION TITLE
Is there a way to Extending Meta or other class for  removing duplication of code

### DISCUSSION THREAD

**Discussion by gin-2026** [storybookjs/storybook#24037] [Help]
# Problem
I feel little boring to write tags or  some type for every my stories. 
 Example from official [document](https://storybook.js.org/docs/vue/writing-docs/autodocs#setup-automated-documentation
).
I always need to write `tags: ['autodocs'],` or other parts. It is so boring. 

```
// Button.stories.ts

// Replace vue3 with vue if you are using Storybook for Vue 2
import type { Meta, StoryObj } from '@storybook/vue3';

import Button from './Button.vue';

const meta: Meta<typeof Button> = {
  component: Button,
  //👇 Enables auto-generated documentation for the component story
  tags: ['autodocs'],
  argTypes: {
    backgroundColor: { control: 'color' },
  },
};

export default meta;
type Story = StoryObj<typeof Button>;

export const Primary: Story = {
  args: {
    primary: true,
    label: 'Button',
  },
};

export const Secondary: Story = {
  args: {
    ...Primary.args,
    primary: false,
  },
};
```

# Question 
Can I make `Meta<typeof Button> extends MetaDefault` or any global configuring for same setting? 
or any similar question of this ? 

# Thank 
Thank you for reading my tiny question. and sorry for bothering you.

**Comment by Jdruwe** [comment]
Also wondering about this, did you find a solution?

**Comment by Jdruwe** [comment]
There is -> https://storybook.js.org/docs/api/csf/csf-next?ref=storybookblog.ghost.io#storyextend
